Updated documentation for v0.6.0 release (#525)
kopia/kopia: site/content/docs/Repositories/_index.md

weight: 40
---

> A repository is a place where Kopia stores all its snapshot data. Kopia can currently connect to the following:

Reviewer: ```suggestion
A repository is a place where Kopia stores its snapshot data. Kopia currently supports the following storage backends:
```

* [Google Cloud Storage](#google-cloud-storage)
* [Azure Blob Storage](#azure)
* [Amazon S3](#amazon-s3) (and compatible)
* [Backblaze B2](#b2)
* [SFTP](#sftp)
* [WebDAV](#webdav)
* [Local storage](#local-storage)

> In addition to that Kopia can connect to a [Repository server](/docs/repository-server/) which acts as a proxy to the data storage.

Reviewer: ```suggestion
In addition, Kopia can connect to a [Kopia Repository Server](/docs/repository-server/) that acts as a proxy for the storage backend.
```
